packages/styles/scss/components/footer/_language-selector.scss

```
@@ -44,6 +44,14 @@
         height: $spacing-09;
         margin-right: 0;
 
+        .#{$prefix}--list-box__menu-icon {
+          right: 0.75rem;
```

```suggestion
          right: $spacing-04;
```
LGTM, could maybe use this token for the `0.75rem` value for this one, but otherwise it's good!